### PR TITLE
fix(telecom.sms): add smpp account order

### PR DESCRIPTION
**Pull request by ghyenne**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/telecom_sms_smpp_ga` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-9906 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
I added the smpp account order

### CODE REVIEW

File: packages/manager/modules/sms/src/sms/order/telecom-sms-order-service.html [hunk 1]
```diff
@@ -4,171 +4,205 @@
...
+        <oui-header data-heading="{{ 'sms_order_title' | translate }}">
+        </oui-header>
 
         <form id="$ctrl.orm" name="$ctrl.orm">
             <div class="row">
-                <div class="col-md-6">
-                    <!-- ACCOUNT -->
-                    <div class="form-group">
+                <!-- ACCOUNT -->
+                <div class="col-sm-12">
```
Comment (mohammed-zahaf) [inline packages/manager/modules/sms/src/sms/order/telecom-sms-order-service.html:13]: i guess we can consider to use `col-12`.
Also, what about for tablets and laptop devices ?

File: packages/manager/modules/sms/src/sms/order/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -1,20 +1,29 @@
 {
   "sms_order_title": "Commander des SMS",
   "sms_order_new_account": "Nouveau Compte",
-  "sms_order_account_label": "Compte SMS : ",
-  "sms_order_credit_label": "Nombre de crédits : ",
-  "sms_order_credit_custom_label": "Quantité désirée : ",
+  "sms_order_account_label": "Sélectionnez le compte à créditer",
+  "sms_order_account_channel_label": "Choisissez le format de compte",
+  "sms_order_account_channel_standard_title": "Standard",
+  "sms_order_account_channel_standard_description": "C'est le compte classique proposé par OVHcloud permettant d'envoyer des SMS unitaires ou par campagne (batch) via le manager ou l'API web OVHcloud.",
```
Comment (mohammed-zahaf) [inline packages/manager/modules/sms/src/sms/order/translations/Messages_fr_FR.json:7]:
```suggestion
  "sms_order_account_channel_standard_description": "C'est le compte classique proposé par OVHcloud permettant d'envoyer des SMS unitaires ou par campagne (batch) via le Manager ou l'API web OVHcloud.",
```

Comment (ghyenne) [reply packages/manager/modules/sms/src/sms/order/translations/Messages_fr_FR.json]: It's applied

File: packages/manager/modules/sms/src/sms/order/telecom-sms-order-service.html (hunk 1 above)
Comment (ghyenne) [reply packages/manager/modules/sms/src/sms/order/telecom-sms-order-service.html:13]: The class "col-12" doesn't works.
The display is responsive for PC and tablets / smartphone.

**Review by mohammed-zahaf — APPROVED** [review]

**Review by vks-ovh — APPROVED** [review]

**Pull request merged**